test_parse.py

```diff
@@ -865,17 +865,17 @@ def test_pickling_bug_110(self):
         # prior to the fix, this would raise an AttributeError
         pickle.dumps(p)
 
-    def test_search_centered_bug_112(self):
-        r = parse.parse("{:^},{:^}", " 12 , 34 ")
-        self.assertEqual(r[1], "34")
-        r = parse.search("{:^},{:^}", " 12 , 34 ")
-        self.assertEqual(r[1], "34")
-
-    def test_search_left_align_bug_112(self):
-        r = parse.parse("{:<},{:<}", "12 ,34 ")
-        self.assertEqual(r[1], "34")
-        r = parse.search("{:<},{:<}", "12 ,34 ")
-        self.assertEqual(r[1], "34")
+    def test_unused_centered_alignment_bug(self):
+        r = parse.parse("{:^2S}", "foo")
+        self.assertEqual(r[0], "foo")
+        r = parse.search("{:^2S}", "foo")
+        self.assertEqual(r[0], "foo")
+
+    def test_unused_left_alignment_bug(self):
+        r = parse.parse("{:<2S}", "foo")
+        self.assertEqual(r[0], "foo")
+        r = parse.search("{:<2S}", "foo")
+        self.assertEqual(r[0], "foo")
```

Comment:
Sorry for the really slow response. Could I trouble you to add a case covering issue #118 please?

Reply:
Sure, no problem.
Actually, issue #118 has two code examples:
```
>>> parse.parse("Column{:d}:{}", "Column 1: Timestep")
<Result (1, ' Timestep') {}>
>>> parse.parse("Column {:d}:{}", "Column 1: Timestep")
<Result (1, ' Timestep') {}>
```
The second one should be fixed by this issue (and it is), but the first one looks like another bug -- if there's no space between `Column` and `{:d}` it shouldn't match anyway, right?

It looks like that's another bug which was introduced when the int parsing was fixed in `bc417de9` -- the regex for `type == 'd'` should be:
```r'\d{w}|[-+]?0[xX][0-9a-fA-F]{w}|[-+]?0[bB][01]{w}|[-+]?0[oO][0-7]{w}'```
instead of:
```r'\d{w}|[-+ ]?0[xX][0-9a-fA-F]{w}|[-+ ]?0[bB][01]{w}|[-+ ]?0[oO][0-7]{w}'```
I've verified that this doesn't break any test case when fixed and am pushing that fix onto this branch as well.

Reply:
Oh I see now that this is not the only issue around this, it also requires fixing:
```
            # allow numbers to be prefixed with a sign
            s = r'[-+]?' + s
```
And that this probably originates to support the case where space is given as the `sign` option, and actually makes test cases break.
Given that `_handle_field` doesn't actually extract `sign` currently, I'll give up on fixing that bug now, since it doesn't break any actual use case for me personally 😉 .

Reply:
Hey @r1chardj0n3s,
I've replied over a month ago and it's still pending. I just wanted to make sure that I didn't miss any step I need to do to set this "request for changes" as "resolved", since it's the first time I'm doing a GitHub PR 😅 .